Sources/OpenAPIRuntime/Conversion/URLExtensions.swift

```diff
@@ -17,7 +17,7 @@ extension URL {
     /// Returns the default server URL of "/".
     ///
     /// Specification: https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#fixed-fields
-    public static let defaultOpenAPIServerURL: Self = {
+    public nonisolated(unsafe) static let defaultOpenAPIServerURL: Self = {
```

Comment:
This would be better fixed by Foundation.URL being Sendable on Linux (it already is on Darwin).

Reply:
For reference: https://github.com/apple/swift-openapi-runtime/pull/107#issuecomment-2180542772